lib/matplotlib/axes/_axes.py

```diff
@@ -2778,10 +2779,16 @@ def errorbar(self, x, y, yerr=None, xerr=None,
         .. plot:: mpl_examples/statistics/errorbar_demo.py
 
         """
-
-        if errorevery < 1:
+        
```

Comment:
I think this might be a bit clearer as

```python
try:
    errorevery, offset = errorevery
except TypeError:
    offset = 0

if errorevery < 1:
    raise
if offset < 0
    raise
```